_episodes/21-summarize.md

```diff
@@ -8,9 +8,12 @@ questions:
 ---
 The final steps in qualifying as an instructor are to:
 
-1.  submit a change to one of our lessons,
-2.  take part in a [discussion session](http://pad.software-carpentry.org/instructor-discussion) with experienced instructors, and
-3.  do a 5-minute live coding demo.
+1.  take part in a [discussion session](http://pad.software-carpentry.org/instructor-discussion) with experienced instructors, and
+2.  do a 5-minute live coding demo.
+3.  make a contribution to a lesson's content, exercises, or instructor's guide by doing **one of the following**:
+  a. submitting a change request to fix an existing issue
+  b. proof-reading a lesson and adding a new issue describing something to be improved
+  c. providing substantive feedback on an existing issue or PR
```

Comment:
I think it's confusing to have these out of order - we really want the live coding demo last.